terraform/infrastructure/load_balancer.tf

```
@@ -146,6 +146,11 @@ resource "aws_lb_listener" "alb_https" {
 
   port     = 443
   protocol = "HTTPS"
+  
+  default_action {
+    type = "forward"
+    target_group_arn = aws_lb_target_group.alb_http.arn
```

In order to match what we have for Traefik (where requests to unrecognized domains get dropped), I'd prefer if we used the ALB's fixed response default action. This suggestion is for a 204 No Content response, in case you need a positive response for health checks, but this could easily be a 404 or even a 5XX status code.

```suggestion
    type             = "fixed-response"
    
    fixed_response {
      content_type = "text/plain"
      status_code  = 204
    }
```